.expeditor/scripts/verify/builder-graph-functional.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Are these setup tasks required for users that want to execute these tests in a studio without expeditor or build-kite?

Reply:
These should all be part of a normal developer setup.  The exception being the build and BLDR_GRAPH_PATH. The reason behind that is that `builder-graph` doesn't fit into the same workflow as the rest of our components; It's not a service so `build-builder` doesn't work and doesn't have `habitat-dev`plan definition. We've been using a `cargo build` workflow external to the studio to develop this package, so the `test/builder-graph/test.sh` script defaults to using the `target/debug/bldr-graph` binary produced by that. `BLDR_GRAPH_PATH` is an escape hatch, primarily for CI,  to tell the driver "no, I actually want to use this other bldr-graph binary", so shouldn't be needed in a normal workflow.